orchagent/port/porthlpr.cpp

```diff
@@ -118,6 +119,12 @@ static const std::unordered_map<std::string, Port::Role> portRoleMap =
     { PORT_ROLE_DPC, Port::Role::Dpc }
 };
 
+static const std::unordered_map<std::string, sai_redis_link_event_damping_algorithm_t> linkEventDampingAlgorithmMap =
```

Comment:
should global names be prefixed with g_ ?

Reply:
Sure, I changed the name to g_linkEventDampingAlgorithmMap.